<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_SourceString">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TargetString">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Output">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
